gluoncv/model_zoo/mobilenetv3.py

```diff
@@ -276,10 +288,10 @@ def mobilenet_v3_large(**kwargs):
         Additional `norm_layer` arguments, for example `num_devices=4`
         for :class:`mxnet.gluon.contrib.nn.SyncBatchNorm`.
     """
-    return get_mobilenet_v3("large", **kwargs)
+    return get_mobilenetv3("large", **kwargs)
 
 
-def mobilenet_v3_small(**kwargs):
+def mobilenetv3_small(**kwargs):
```

Comment:
keep `mobilenet_v3_small` and `mobilenet_v3_large` so that it is consistent with the name called in `model_zoo.py`